### DISCUSSION TITLE
[Bug]: Can't Preview .esp .svg images and all videos formate are not working

### DISCUSSION THREAD

**Discussion by Abdelrhman02** [pimcore/pimcore#16726] [Q&A / Support]
### Pimcore version

11

### Steps to reproduce

Upload videos and images in assets and open them the preview is not working

 php -i | grep magic
/etc/php/8.1/cli/conf.d/20-imagick.ini,
libmagic => 540
imagick
imagick module => enabled
imagick module version => 3.7.0
imagick classes => Imagick, ImagickDraw, ImagickPixel, ImagickPixelIterator, ImagickKernel
Imagick compiled with ImageMagick version => ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
Imagick using ImageMagick library version => ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
imagick.allow_zero_dimension_images => 0 => 0
imagick.locale_fix => 0 => 0
imagick.progress_monitor => 0 => 0
imagick.set_single_thread => 1 => 1
imagick.shutdown_sleep_count => 10 => 10
imagick.skip_version_check => 1 => 1

 php -i | grep AVIF
AVIF Support => enabled

php -i | grep WebP
WebP Support => enabled

sudo snap install chromium
snap "chromium" is already installed, see 'snap help refresh'
sudo snap refresh
All snaps up to date.
sudo apt install chromium-browser
Reading package lists... Done
Building dependency tree       
Reading state information... Done
chromium-browser is already the newest version (1:85.0.4183.83-0ubuntu0.20.04.3).

### Actual Behavior

The preview is not working. 

### Expected Behavior

preview videos and images should work.

**Comment by ghost** [comment]
@Abdelrhman02 Sorry, but we cannot reproduce it on our demo environments. Uploading images and videos works for us. Maybe the community can help?

**Comment by EliShklovsky** [comment]
On Debian you can run
sudo apt-get install libmagickcore-6.q16-6-extra
This package adds support for SVG, WMF, OpenEXR, DjVu and Graphviz to MagickCore
